scripts/ci-entrypoint.sh

```diff
@@ -207,6 +211,9 @@ install_addons() {
             --set-string cloudControllerManager.clusterCIDR="${CCM_CLUSTER_CIDR}"
     fi
 
+    export -f wait_for_nodes
+    timeout --foreground 1800 bash -c wait_for_nodes
```

Comment:
Are we worried that waiting for the CCM install will make it more likely we'll hit this timeout?
Given that the timeout is 30 mins I would hope not but I wanted to ask...

Reply:
I don't think I understand the question, could you please expand?

Reply:
I've verified this doesn't cause issues in local testing